lib/adapter/freshheads/LoadReferencedFilesAdapter.ts

```diff
@@ -26,19 +25,15 @@ export default class LoadReferencedFilesAdapter implements Adapter {
         builderConfig: BuilderConfig,
         next: NextCallback
     ) {
-        if (builderConfig.env === Environment.Dev) {
-            validateIfRequiredModuleIsInstalled(
-                'LoadReferencedFilesAdapter',
-                'file-loader',
-                '6.0.0'
-            );
-        }
+        const isDevelopment = builderConfig.env === Environment.Dev;
 
         const rule: RuleSetRule = {
             test: this.config.test,
-            loader: 'file-loader',
-            options: {
-                name: '[name].[contenthash].[ext]',
+            type: 'asset/resource',
+            generator: {
+                filename: isDevelopment
+                    ? '[name].[hash][ext][query]'
+                    : '[hash][ext][query]',
```

Comment:
Waarom dit verschil? Waarom in prod-environment niet ook de `name`? We maken het nu misschien onnodig complex?

Reply:
hmm ja is mss ook wel zo, de default is `[hash][ext][query]` maar dacht dat het in dev wel fijn is om te zien wat er geladen word. Maar het is inderdaad geen geheim dus kan ook prima op prod. De andere assets zoals Js hebben dat in prod weer niet. Vandaar dat ik hier voor was gegaan.

Zal het in volgende release aanpassen.